plot-doc/plot/scribblings/utils.scrbl

```diff
@@ -370,7 +370,7 @@ Returns @racket[#t] if every element of @racket[v] is @racket[rational?].
 @;----------------------------------------------------------------------------------------------------
 @subsection[#:tag "math.intervals"]{Intervals and Interval Functions}
 
-@defstruct[ivl ([min real?] [max real?])]{
+@defstruct[ivl ([min (or/c real? #f)] [max (or/c real? #f)])]{
```

Comment:
Since `rectangles` is using `inf` values now, I propose un-doing this change to the docs.

[[ I have no idea why `ivl`s can contain `#false` internally. So as long as this PR is not adding a new API that allows `#false`, I think we better keep it a secret that `(ivl #false #false)` is allowed. ]]